tests/testthat/test-spClipPoly.R

```diff
@@ -1,26 +1,33 @@
 
 test_that("Clip polygon works as expected", {
 
-  # Setup data 
-  USAco <- geodata::gadm(country = "USA", level = 2, path = tempdir())
-  WYco <- USAco[USAco$NAME_1 == "Wyoming"]
-
+  # Setup Admin Boundary to Clip
   WYbhfn <- system.file("extdata",
                         "sp_data/WYbighorn_adminbnd.shp",
                         package = "FIESTA")
 
   WYbh <- spImportSpatial(WYbhfn)
 
-  # Test
-  clipped <- spClipPoly(polyv = WYco, clippolyv = WYbh, areacalc = TRUE)
+  # Setup District Boundary and subset to clip by
+  WYbhdistfn <- system.file("extdata",
+                            "sp_data/WYbighorn_districtbnd.shp",
+                            package = "FIESTA")
+
+  WYbhdist <- spImportSpatial(WYbhdistfn)
+
+  subset_district <- WYbhdist[WYbhdist$DISTRICTNU == "01",]
```

Comment:
woah! subset an SF object with a comma at the end! Nice!

Reply:
Will never forget that now